osu.Game/Beatmaps/BeatmapSetOnlineStatusExtensions.cs

```diff
@@ -3,6 +3,12 @@
 
 namespace osu.Game.Beatmaps
 {
+    public static class BeatmapSetOnlineStatusExtensions
+    {
+        public static bool HasPerformancePoints(this BeatmapSetOnlineStatus status)
```

Comment:
Actually not sure about the name once I'm reading the usages there, I guess `ShowsPerformancePoints` would suffice.

Reply:
if it's an extension method of `BeatmapSetOnlineStatus`, something like `status.GrantsPerformancePoints()` makes more sense.